src/integrators/integrator_utils.jl

```diff
@@ -296,6 +296,187 @@ function step_reject_controller!(integrator,alg::PredictiveControllerAlgs)
   end
 end
 
+
+@inline function stepsize_controller!(integrator,alg::ExtrapolationMidpointDeuflhard)
+  # Dummy function
+  # ExtrapolationMidpointDeuflhard's stepsize scaling is stored in the cache;
+  # it is computed by  stepsize_controller_internal! (in perfom_step!) resp. stepsize_predictor!
+  # (in step_accept_controller! and step_reject_controller!)
+  zero(typeof(integrator.opts.qmax))
+end
+
+function stepsize_controller_internal!(integrator,alg::ExtrapolationMidpointDeuflhard)
+  # Standard stepsize controller
+  # Compute and save the stepsize scaling based on the latest error estimate of the current order
+  if iszero(integrator.EEst)
+    q = inv(integrator.opts.qmax)
+  else
+    # Update gamma and beta1
+    integrator.opts.beta1 = typeof(integrator.opts.beta1)(1 // (2integrator.cache.n_curr + 1))
+    integrator.opts.gamma = typeof(integrator.opts.gamma)(1 // 4)^integrator.opts.beta1
+    # Compute new stepsize scaling
+    qtmp = integrator.EEst^integrator.opts.beta1 / integrator.opts.gamma
```

Comment:
I see why you got rid of beta2. Something that would be worthwhile later would be to try and make this use PI instead of P, since step stabilization will help a method based on explicit RKs take larger steps.

Reply:
But hmm, yes anything based on a work estimate is hard to stabilize.